src/tests/system/tests/test_ldap.py

```diff
@@ -502,3 +515,134 @@ def test_ldap__password_change_no_grace_logins_left(
 
     rc, _, _, _ = client.auth.parametrize(method).password_with_output("user1", "Secret123")
     assert rc == expected, err_msg
+
+
+@pytest.mark.parametrize("Operation", ["Add", "Replace"])
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup__add_dn_membernisnetgroup(client: Client, ldap: LDAP, Operation: str):
+    """
+    :title: SSSD processes 'memberNisNetgroup' attribute is the DN of a group.
+    :setup:
+        1. Create users, groups.
+        2. Create a new netgroup called QAUsers and add a member (ng9000) to QAUsers
+        3. Create another netgroup named DEVUsers and add a member (ng9005) to DEVUsers
+        4. Modify the DEVUsers netgroup to replace its members with the members of QAUsers.
+        5. Start sssd
+    :steps:
+        1. Retrieve all members of the DEVUsers netgroup.
+        2. Confirm that the member directly added to DEVUsers is present.
+        3. Confirm that the member from QAUsers is now part of DEVUsers.
+    :expectedresults:
+        1. All members should be retrieved
+        2. Members directly added to DEVUsers is present.
+        3. Members from QAUsers is now part of DEVUsers.
+    :customerscenario: False
+    """
+    ou = ldap.ou("Netgroup")
+    create_users(ldap)
+
+    qa_users = ldap.netgroup("QAUsers", basedn=ou).add()
+    qa_users.add_member(host="testhost1", user="ng9000", domain="ldap.test")
+
+    dev_users = ldap.netgroup("DEVUsers", basedn=ou).add()
+    dev_users.add_member(host="testhost5", user="ng9005", domain="ldap.test")
+    if Operation == "Replace":
+        ldap.ldap.modify(dev_users.dn, replace={"memberNisNetgroup": qa_users.dn})
+    else:
+        ldap.ldap.modify(dev_users.dn, add={"memberNisNetgroup": "QAUsers"})
+    client.sssd.start()
+
+    member = client.tools.getent.netgroup("DEVUsers").members
+    assert "(testhost5, ng9005, ldap.test)" in member
+    assert "(testhost1, ng9000, ldap.test)" in member
+
+
+@pytest.mark.parametrize(
+    "user, domain, expected",
+    [("samplehost", "samplehost.domain.com", "(samplehost,-,samplehost.domain.com)"), ("ng9006", "", "(-,ng9006,)")],
+)
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup__host_and_domain(client: Client, ldap: LDAP, user: str, domain: str, expected: str):
```

Comment:
I don't think the value in the netgroup being looked up is essential. Having the value be a user, or a host is more of a test of the directory if it accepts that value? 

@ikerexxe does SSSD expect a particular type of value, or are there any constraints when looking up identities? 

If it doesn't, then I think this test should be dropped.

Reply:
What is the resolution? Are we dropping this or keeping it? Please refrain from resolving issues that are not resolved yet.